plugins/GrapesJsBuilderBundle/Assets/library/js/builder.service.js

```diff
@@ -275,6 +276,27 @@ export default class BuilderService {
   getEditor() {
     return this.editor;
   }
+
+  // https://github.com/artf/grapesjs-mjml/issues/193
+  overrideCustomRteDisable() {
+    const richTextEditor = this.editor.RichTextEditor;
+
+    if (!richTextEditor) {
+      throw new Error('No RichTextEditor found');
```

Comment:
Is it necessary to throw an error? I'm not sure if this can happen but can we for example just return early here and log it as error instead? The error is not caught anywhere, right?

Reply:
@escopecz right, good idea. Instead, I changed the error throwing to console.error. Please check again.